server.js

```diff
@@ -81,11 +82,11 @@ wsServer.on('connection', function connection(ws, request) {
                     break
                 case 'CreateRoom':
                     console.log('CreateRoom Request received from client %o', jsonObj)
-                    roomId = util.createRoom(lobby, jsonObj.playerId, jsonObj.playerName, jsonObj.playerAvatar, jsonObj.maxPlayersPerRoom)
+                    roomId = util.createRoom(lobby, jsonObj.playerId, jsonObj.playerName, jsonObj.playerAvatar, jsonObj.maxPlayersPerRoom, jsonObj.playerTags)
```

Comment:
App dependent tags?

Reply:
Exactly, When the player connects to the server, they can pass tags <KeyValue List> to the server.
The Hashtable to Dictionary in DistributedMatchingEngine.cs (tags) made the c# implementation super easy for me. 
https://github.com/mobiledgex/edge-multiplay-unity-client/pull/9